package.json

```diff
@@ -8,15 +8,20 @@
     "url": "https://github.com/Caltech-IPAC/firefly.git"
   },
   "dependencies": {
-    "babel-core": "^6.26.0",
-    "babel-loader": "^7.1.2",
+    "@babel/core": "^7.5.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-json-strings": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/plugin-syntax-import-meta": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.0.0",
```

Comment:
The above plugins are needed now?  Do you know what is requiring it?

Reply:
These plugins are used instead of stage-3 plugin. As far as I understood from:
https://github.com/babel/babel-upgrade
(it is mentioned at the end of that link).

Reply:
I don't think we are using all these. I think we where using stage-3 there before for object expanding (ie.`{...x, {y:1}}`) when it is not part of the standard.

Let me look at each of these.

Reply:
Cool, I will take them out.

Reply:
Take them out and test, make sure nothing breaks.

Reply:
Fixed, thanks!